### DISCUSSION TITLE
Option to only include certain authors

### DISCUSSION THREAD

**Discussion by resucutie** [ThioJoe/YT-Spammer-Purge#178] [Ideas]
Doing a scan on a video made by [Dani](https://www.youtube.com/watch?v=K0X5A6aURms) i saw a lot of false alerts, and well excluding them manually will be a pain in the ass, so my suggestion is to add an option to include specific people

It would work simmilarly to the exclude option, but it excludes every commentary except the specified ones

**Comment by ThioJoe** [comment]
Yea I'll probably add a way to make a whitelist of channel IDs. 

And I see what you're saying about the false positives for that, looks like it's because the channel's username is pretty short, so the program is matching anyone with that in theirs, including 'Daniel', etc.

**Reply by resucutie** [reply]
Yeah, it is what is happening lol
![image](https://user-images.githubusercontent.com/27961224/147172072-49a8766f-2184-4111-936a-3f9751271105.png)

**Comment by ThioJoe** [comment]
Just put up a new release which should greatly improve those false positives. There might still be a few though, I'll have to see about improving it further.
